proxy/src/main/java/pt/webdetails/cda/CdaUtils.java

```diff
@@ -56,20 +55,33 @@
 public class CdaUtils {
   private static final Log logger = LogFactory.getLog( CdaUtils.class );
 
+  private static final int HTTP_RESPONSE_OK = 200;
```

Comment:
An alternative to this constant would be something like `ClientResponse.Status.OK.getStatusCode()`